shell/pages/c/_cluster/explorer/EventsTable.vue

```diff
@@ -43,14 +43,19 @@ export default {
         ...STEVE_NAME_COL,
         defaultSort: false,
       },
-      headerFromSchemaColString('First Seen', schema, this.$store.getters, true),
-      {
-        ...headerFromSchemaColString('Last Seen', schema, this.$store.getters, true),
-        defaultSort: true,
-      },
-      headerFromSchemaColString('Count', schema, this.$store.getters, true),
     ];
 
+    if (schema) {
```

Comment:
Not sure if we want to remove these headers or assign hardcoded values when schema is null